frontend/src/organizations/components/OrganizationDetails.js

```diff
@@ -114,6 +114,15 @@ const OrganizationDetails = props => {
             <dd>Email <a href="mailto:lcfs@gov.bc.ca?subject=TFRS Address Update">lcfs@gov.bc.ca</a> to update address information.</dd>
           </dl>
         </div>
+        {/* {props.organization.organizationAddress &&
```

Comment:
If we don't need this commented out code, then best to remove it.

Reply:
yeah.. removed it.